test/wpa_encode_test.exs

```diff
@@ -42,7 +42,7 @@ defmodule WpaEncodeTest do
     assert Messages.encode(:ADD_NETWORK) == "ADD_NETWORK"
     assert Messages.encode({:SET_NETWORK, 1, :ssid, "SSID"}) == "SET_NETWORK 1 ssid \"SSID\""
     assert Messages.encode({:SET_NETWORK, 1, :psk, "SSID"}) == "SET_NETWORK 1 psk \"SSID\""
-    assert Messages.encode({:SET_NETWORK, 1, :key_mgmt, "SSID"}) == "SET_NETWORK 1 key_mgmt \"SSID\""
+    assert Messages.encode({:SET_NETWORK, 1, :key_mgmt, "SSID"}) == "SET_NETWORK 1 key_mgmt SSID"
```

Comment:
I think we should change these assertions to validate the actual things we expect, like this:

```elixir
assert Messages.encode({:SET_NETWORK, 1, :key_mgmt, :"WPA-PSK"}) == "SET_NETWORK 1 key_mgmt WPA-PSK"
assert Messages.encode({:SET_NETWORK, 1, :key_mgmt, "WPA-PSK"}) == "SET_NETWORK 1 key_mgmt WPA-PSK"
assert Messages.encode({:SET_NETWORK, 1, :key_mgmt, :NONE}) == "SET_NETWORK 1 key_mgmt NONE"
assert Messages.encode({:SET_NETWORK, 1, :key_mgmt, "NONE"}) == "SET_NETWORK 1 key_mgmt NONE"
```